README.md

```diff
@@ -34,6 +34,23 @@ it, simply add the following line to your Podfile:
 pod "ViewAnimator"
 ```
 
+### Carthage
+
+You can install Carthage with [Homebrew](http://brew.sh/) using the following command:
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+To integrate ViewAnimator into your Xcode project using Carthage, specify it in your `Cartfile`:
+
+```ogdl
+git "hamzaansari09/ViewAnimator" "1.0.2"
```

Comment:
Also I think this is not correct, the owner of the repo is yourself instead of the original author of the project, which is who  will be hosting the whole project, isn't it?

Reply:
Of course, I have done it for testing. It's need to be updated to the owner's repo once he merge and create the new tag.